### PR TITLE
[PLAY-555] Fixed Body Content disappearing behind Footer for Dialog

### PR DESCRIPTION
**Pull request by nidaqg**
## Before:
Screenshot 2023-01-25 at 4 01 01 PM

## After:
Screenshot 2023-01-25 at 3 54 47 PM

#### Breaking Changes

No breaking changes

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-555)

#### How to test this

Click on full height variant in 'scrollable' doc example. Scroll to the bottom to see text is no longer covered by footer.  Test both Rails and React.

#### Checklist:

- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [ ] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

**Review by jasperfurniss — APPROVED** [review]

File: playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss [hunk 1]
```diff
@@ -227,6 +227,9 @@
 				height: 100%;
 				max-height: 100%;
 				max-width: none;
+				.dialog_body {
+					padding-bottom: $space_xl * 2 !important;
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss:231]: Does this still work with our Global Spacing props. 
For example, if I use a complex dialog... and have a `<Dialog.Body paddingBottom="xs" />` which `!important` wins out?

Comment (nidaqg) [reply playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss:231]: Good point, I'll pull down the branch again and test it out @jasperfurniss

Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss:231]: Great! Let me think through this one for a minute, and get back to you.

Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss:231]: Yes. Makes sense. I did notice the small scope, but still a little bit of a code smell, when our global props don't take specificity priority over something... Let me see if i can brainstorm a workaround.

**Review by jasperfurniss — APPROVED** [review]

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**